node-notes-cli/add.js

```diff
@@ -0,0 +1,22 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+async function addNote() {
+  try {
+    const fileData = await readFile('data.json');
```

Comment:
For maintenance purposes, it would be good to move the read and write operations into their own file that you import.